featurevis/featurevis.py

```diff
@@ -131,7 +132,7 @@ def visualize_filter(image, model, layer, filter_index, iterations,
     print('>> 100 %')
     # Decode the resulting input image
     image = imgs.deprocess_image(image[0].numpy())
-    return loss, image
+    return loss, image, layer, filter_index
```

Comment:
Why would we return layer and filter_index? We already provide this to the function as a parameter, so the other end should know about this, no?

Reply:
You are right. I will implement the variables in the main function and use them for the file name generation there.